tagsets/query.php

```diff
@@ -147,11 +147,11 @@ function pos_prototype($icon='bars') {
             );
         }
         echo "<script type='text/javascript'>var Ptypes = ";
-        echo json_encode($tmp);
+        echo json_encode(fix_utf8($tmp));
```

Comment:
Would not passing `JSON_INVALID_UTF8_IGNORE` param to `json_encode` do the same (only available from php7.2 though)?